docker/docker-compose.dev.yml

```diff
@@ -24,6 +24,7 @@ services:
       - ${UPLOAD_LOCATION}/photos/upload:/usr/src/app/upload/upload
       - /usr/src/app/node_modules
       - /etc/localtime:/etc/localtime:ro
+      - ${UPLOAD_LOCATION}/external:/usr/src/app/external
```

Comment:
Is that supposed to be in this PR?

Reply:
Nope, accidentally made it in with the other changes 👀